.github/workflows/focal-ci.yaml

```
@@ -0,0 +1,28 @@
+name: focal-ci
+on:
+  push:
+    branches:
+      - galactic
+  pull_request:
+    branches:
+      - galactic
+jobs:
+  focal:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        gazebo-version:
+          - "citadel"  # libsdformat9
+          - "dome"  # libsdformat10
+          - "ediface"  # libsdformat11
```

Dome and Edifice are EOL, I really discourage adding support or CI for them. I only suggested adding Edifice to the `galactic` branch because that's the official version combination and we're committed to supporting that until Galactic itself EOLs.

```suggestion
          - "edifice"  # libsdformat11
```